.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   env: {
     node: true,
     browser: true,
+    es6: true,
```

Comment:
Otherwise the linter complains about `Set` not being defined:

```
✖ eslint --fix:

/Users/rudolfs/work/radicle-upstream/ui/Screen/DesignSystemGuide.svelte
  63:12  error  'Set' is not defined  no-undef
```